src/@PhreeqcRM/PhreeqcRM.m

```diff
@@ -167,7 +171,7 @@ n	A cell number in the PhreeqcRM numbering system (0 <= n < RM_GetChemistryCellC
             %}
             ncomps = obj.RM_FindComponents();
             components = cell(ncomps, 1);
-            s_name = '000000000000000000000000000'; % untill I find a more elegant solution
+            s_name = blanks(27); % untill I find a more elegant solution
```

Comment:
in the next version, we can have a fixed value for creating these blanks. It can be a property of the Phreeqc class or an environment variable.

Reply:
In the C code, they allocate 100 characters for each string (char str[100];). Perhaps we can follow the same approach.